docs/backup-and-restore.md

```diff
@@ -9,11 +9,13 @@ DC/OS Apache Cassandra supports backup and restore from S3 storage for disaster
 
 Cassandra takes a snapshot of your tables and ships them to a remote location. Once the snapshots have been uploaded to a remote location, you can restore the data to a new cluster, in the event of a disaster, or restore them to an existing cluster, in the event that a user error has caused a data loss.
 
+**Note:** Backup and restore is not guaranteed to work across arbitrary versions of the Cassandra service.  For example restoring a backup from a 1.0.16 cluster to a 1.0.21 cluster will not work.  It is recommended that backups be restored to clusters of the same version from which they were created.
```

Comment:
@joerg84 I believe the use of backups for upgrades is only for a specific version jump.  (And may not work to any arbitrary version in the future...  we might want to constrain that.)